src/main/java/org/ladocuploader/app/inputs/LaDigitalAssister.java

```diff
@@ -150,6 +165,21 @@ public class LaDigitalAssister extends FlowInputs {
   
   private String personalSituationDisability;
 
+
+  @Money
+  private String hourlyWage;
+
+  @Range(min=1, max=24)
```

Comment:
@lkemperman-cfa - why is the limit per week 24 hours?

Reply:
Good catch - this is a mistake idk what the limit should actually be or if there needs to be one